circuits/cpp/src/aztec3/circuits/kernel/private/.test.cpp

```diff
@@ -537,14 +537,32 @@ TEST(private_kernel_tests, native_deposit)
     NT::fr const& asset_id = 1;
     NT::fr const& memo = 999;
 
-    auto const& private_inputs = do_private_call_get_kernel_inputs_init(false, deposit, { amount, asset_id, memo });
+    auto private_inputs = do_private_call_get_kernel_inputs_init(false, deposit, { amount, asset_id, memo });
```

Comment:
might be worth initialising the `encrypted_logs_hash`, `encrypted_log_preimages_length` in the `do_private_call_get_kernel_inputs_init` method itself